src/edu/ucsb/nceas/metacat/startup/MetacatInitializer.java

```diff
@@ -89,14 +89,13 @@ public void contextInitialized(ServletContextEvent sce) {
             // Check to see if the user has requested to bypass configuration
             // (dev option) and check see if metacat has been configured.
             // If both are false then stop the initialization
-            if (!ConfigurationUtil.bypassConfiguration() &&
-                                    !ConfigurationUtil.isMetacatConfigured()) {
+            if (!Boolean.parseBoolean(System.getenv("METACAT_IN_K8S"))
```

Comment:
this "should" work, but someone needs to test it. I can do that when I get back, if you want? If so, could you please create an Issue, assign me, and send me the link?